### ISSUE TITLE
Double job?

### ISSUE THREAD

**Issue by Dokman** [renzuzu/renzu_hud#17]
Can be added the double job onto the info?

**Comment by renzuzu** [comment]
what is the double job script? i need a base logic to implement it

**Comment by Dokman** [comment]
it's a secondary job, for example i can be a policeman and a drug dealer (in-game obviously)

**Comment by renzuzu** [comment]
yes thats why i ask which script is the double job, default esx does not have setjob 2

**Comment by Dokman** [comment]
but i can be scripted and are always the same "setjob" but instead of setjob is setjob2, i use it for gangs, that can work with another job too
